### ISSUE TITLE
TypeError: sequence item 0: expected str instance, bytes found

### ISSUE THREAD

**Issue by dmoisset** [machinalis/protobuf-python3#35]
This error is produced by the following two tests:

```
ERROR: testMergeGolden (google.protobuf.internal.text_format_test.TextFormatTest)
ERROR: testMergeGoldenExtensions (google.protobuf.internal.text_format_test.TextFormatTest)
```

**Comment by dmoisset** [comment]
fixed

**Comment by dmoisset** [comment]
This reappeared after a few fixes

**Comment by juannorris** [comment]
Fixed in 06101937239a81ca69b180ec0fdfbede994c23dd.
